src/index.ts

```diff
@@ -426,6 +435,23 @@ export class LoggingBunyan extends Writable {
    * @param callback The callback supplied by Writable.write
    */
   _writeCall(entries: Entry | Entry[], callback: Function) {
+    // First create instrumentation record if it is never written before
+    if (!setInstrumentationStatus(true)) {
```

Comment:
nit: modifying the instrumentationStatus and acting on the old state in the same line could be confusing. You might want to break this up into two lines for readability: 

```
let alreadyWritten =  setInstrumentationStatus(true);
if (!alreadyWritten) {
...
```

Reply:
Sure thing - I usually avoid extra parameters, thats why I also added a comment on top of the line. But if it confusing, I can definitely make an exception